website/src/controllers/user.js

```diff
@@ -117,12 +117,26 @@ api.score = function(req, res, next) {
     }, saved.toJSON().stats));
 
     // Webhooks
+
+    // Select character data to send
+    var userData = _.pick(user.toJSON(), ['_id', '_tmp', 'stats']); // user.toJSON to copy-by-value
+    userData.stats.toNextLevel = shared.tnl(user.stats.lvl);
+    userData.stats.maxHealth = shared.maxHealth;
+    userData.stats.maxMP = user._statsComputed.maxMP;
+
+    // for each webhook
     _.each(user.preferences.webhooks, function(h){
       if (!h.enabled || !validator.isURL(h.url)) return;
       request.post({
         url: h.url,
         //form: {task: task, delta: delta, user: _.pick(user, ['stats', '_tmp'])} // this is causing "Maximum Call Stack Exceeded"
-        body: {direction:direction, task: task, delta: delta, user: _.pick(user, ['_id', 'stats', '_tmp'])}, json:true
+        body: {
+          direction: direction, // direction of change
```

Comment:
:+1: on breaking these out into separate lines, but I don't think the comments are needed.